design/proposals/identity_support.md

```diff
@@ -0,0 +1,254 @@
+| conjur-cli-go | Happy Path Integration Test | Given an AuthnOIDC V2 instance is properly configured to authenticate with an Identity instance | When a user properly authenticates with AuthnOIDC V2 through the Go CLI | Then the user can use the CLI to perform authorized actions, like policy load and secret retrieval. | See [Conjur Go CLI's Happy Path Integration Test for Okta](https://github.com/cyberark/conjur-cli-go/blob/master/cmd/integration/oidc_integration_test.go#L146) |
+| conjur-ui     | Happy Path Integration Test | Given an AuthnOIDC V2 instance is properly configured to authenticate with an Identity instance | When a user properly authenticates with AuthnOIDC V2 through the UI | Then the user is logged in to the UI | See [Conjur UI's Happy Path Integration Test for Okta](https://github.com/conjurinc/conjur-ui/blob/caf72e0cd8acb3dd3d9c8ab6556a42f4aec7d7fa/features/okta.feature) |
+
+## Documentation
+[//]: # "Add notes on what should be documented in this solution. Elaborate on where this should be documented, including GitHub READMEs and/or official documentation."
+
+- [OpenID Connect (OIDC) Authenticator](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/OIDC/OIDC.htm?tocpath=Integrations%7COpenID%20Connect%20(OIDC)%20Authenticator%7C_____0)
+
+  Limitations section lists Okta as the only available provider. This list needs to be updated.
+
+- [OIDC Authenticator for Conjur UI and Conjur CLI](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/OIDC/OIDC-for-UI-and-CLI.htm#OIDC%C2%A0Aut2) 
+
+  There are a lot of points in the documentation where we describe a service or
+  configuration based on Okta, but we could change this to Identity to be internally consistent.
+
+## Security
+[//]: # "Are there any security issues with your solution? Even if you mentioned them somewhere in the doc it may be convenient for the security architect review to have them centralized here"
+
+Given the limited scope of this effort, and given the rigorous consideration of
+security issues before and during the initial implementation of AuthnOIDC V2, I
+have not encountered any new security concerns.
```

Comment:
I agree on Identity as a standalone provider not having additional security concerns. 

I have a very minor concern about bleed through when Identity & another OIDC provider are both configured - I don't see any way that I think it's likely to happen right now but when we do the implementation review, let's double check that authentication with one has no path to affect logging in via the other.  I don't think it will - there should be enough safeguards in the OIDC protocol - but there's enough of a nudge around that's it worth revisiting the flow when we do the final review just to make sure we didn't miss anything.

Reply:
Sounds good - I'll add a note to capture this.